sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/StaticFieldInitializerOrder.cs

```diff
@@ -49,7 +49,8 @@ protected override void Initialize(SonarAnalysisContext context)
                 {
                     var fieldDeclaration = (FieldDeclarationSyntax)c.Node;
                     var variables = fieldDeclaration.Declaration.Variables;
-                    var classDeclaration = fieldDeclaration.FirstAncestorOrSelf<ClassDeclarationSyntax>();
+                    var typeDeclaration = fieldDeclaration.FirstAncestorOrSelf<TypeDeclarationSyntax>(
```

Comment:
the relevant change is this line

all the other changes in the file are refactorings
- change name from *Class* to *Type*
- use `TypeDeclarationSyntax` instead of `ClassDeclarationSyntax`